<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="AllStackWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="MainStackWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="loginGroupBox">
          <property name="title">
           <string>Log in</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>221</width>
             <height>22</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelUsername">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditUsername"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>221</width>
             <height>22</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelPassword">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPassword">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget_3">
           <property name="geometry">
            <rect>
             <x>89</x>
             <y>140</y>
             <width>138</width>
             <height>25</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="comboBoxUserAdmin">
              <item>
               <property name="text">
                <string>User</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Admin</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonLogin">
              <property name="text">
               <string>Log in</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="ErrorLable">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>221</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Error</string>
           </property>
          </widget>
          <zorder>layoutWidget_2</zorder>
          <zorder>layoutWidget_3</zorder>
          <zorder>layoutWidget</zorder>
          <zorder>ErrorLable</zorder>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="HomeStackWidget">
       <widget class="QPushButton" name="pushButtonAddTaluka">
        <property name="geometry">
         <rect>
          <x>724</x>
          <y>30</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Taluka</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSchoolManager">
        <property name="geometry">
         <rect>
          <x>724</x>
          <y>70</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>School Manager</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="SchoolsManager">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBoxSchoolManagerMain">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableView" name="tableViewSchools"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxManager">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButtonAddSchool">
                <property name="text">
                 <string>Add School</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonEditSchool">
                <property name="text">
                 <string>Edit School</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonRemoveSchool">
                <property name="text">
                 <string>Remove School</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxSchoolManagerControl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonBack">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionChooseDatabase"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAddUser"/>
    <addaction name="actionAddAdmin"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionAddUser">
   <property name="text">
    <string>Add User</string>
   </property>
  </action>
  <action name="actionAddAdmin">
   <property name="text">
    <string>Add Admin</string>
   </property>
  </action>
  <action name="actionChooseDatabase">
   <property name="text">
    <string>Choose Database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
